src/Microsoft.AspNetCore.Mvc.Core/Routing/ConsumesMatcherPolicy.cs

```diff
@@ -173,7 +173,7 @@ public PolicyJumpTable BuildJumpTable(int exitDestination, IReadOnlyList<PolicyJ
             return new ConsumesPolicyJumpTable(exitDestination, ordered);
         }
 
-        private int GetScore(MediaType mediaType)
+        private int GetScore(in MediaType mediaType)
```

Comment:
perf? (curious)

Reply:
I have an analyzer (installed via a VSIX) that warns about suboptimal struct usage 😆